fiwaremurano/meta/io.murano/Resources/support.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+same_uuid() {
+  curl 'http://169.254.169.254/openstack/latest/meta_data.json' --output /etc/metadata.json
+  if [ -f "/etc/metadata.json" ]
+  then
+    if grep -Po '"uuid":.*?[^\\]",' /etc/metadata.json | grep `cat /etc/fiware-support/uuid`; then
```

Comment:
No, my question is about the file /etc/fiware-support/uuid, I can imagine that this file is generated by activate_support_account but if for some reason or error it is not generated this sentence produce an error and do not go into the if code.

Are you sure that the file is generated on all posible cases?